### ISSUE TITLE
Loopia - Add-LoopiaTXTRecord broken

### ISSUE THREAD

**Issue by AlexanderRydberg** [rmbolger/Posh-ACME#424]
I guess that Loopia has changed something on the api. This is where it throws exception when running:  
`New-PACertificate -Domain $certNames -DnsPlugin Loopia -PluginArgs $loopiaParams -PfxPass $pfxPass -AcceptTOS -debug`

```
DEBUG: Adding TXT record for _acme-challenge.YYYY.domain with value XXXXXX

Confirm
Continue with this operation?
[Y] Yes  [A] Yes to All  [H] Halt Command  [S] Suspend  [?] Help (default is "Y"):
VERBOSE: POST https://api.loopia.se/RPCSERV with -1-byte payload
VERBOSE: received -1-byte response of content type text/html; charset=UTF-8
Param must contain a value
Submit-ChallengeValidation : Error adding Loopia TXT record: Param must contain a value
At C:\Users\XXX\Documents\WindowsPowerShell\Modules\Posh-ACME\4.13.0\Public\New-PACertificate.ps1:238 char:9
+         Submit-ChallengeValidation
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : OperationStopped: (Error adding Lo...contain a value:String) [Submit-ChallengeValidation], RuntimeExceptio
   n
    + FullyQualifiedErrorId : Error adding Loopia TXT record: Param must contain a value,Submit-ChallengeValidation
    
```

By the looks of it, Add-LoopiaTXTRecord seems to fail. I think all the other ones work. 

Tried to debug "C:\\Users\\XXX\\Documents\\WindowsPowerShell\\Modules\\Posh-ACME\\4.13.0\\Plugins\\Loopia.ps1" but failed to see which param that was missing.

**Comment by rmbolger** [comment]
Hey @AlexanderRydberg, thanks for reaching out. This is the second provider recently that changed their API and broke stuff. I see you already found and submitted a fix via acme.sh. Thanks for that as well. I'll take a look and get it in asap.

**Comment by rmbolger** [comment]
This is now fixed in 4.13.1. Thanks again.
